creator-node/src/snapbackSM/secondarySyncHealthTracker.js

```diff
@@ -169,17 +169,42 @@ const Getters = {
       logger.error(`SecondarySyncHealthTracker - getSyncRequestOutcomeMetrics() Error || ${e.message}`)
       return {}
     }
+  },
+
+  /**
+   * Returns single int representing SyncRequestOutcome for secondary, wallet, syncType, date=today, and Outcome=Failure
+   * Only one redis key should exist for above params, but takes 1st value if multiple are found
+   */
+  async getSecondaryUserSyncFailureCountForToday (secondary, wallet, syncType) {
+    const resp = await this.getSyncRequestOutcomeMetrics({
```

Comment:
can we update `this` to `Getters`? to follow the rest of the file

Reply:
done in upcoming PR, good callout